src/util.rs

```diff
@@ -1,6 +1,29 @@
 /// Common helpers
 use anyhow::{anyhow, Result};
 
+use ring::digest::{Context, Digest, SHA256};
```

Comment:
Note: `ring` is [not a friendly crate as a dependency](https://github.com/briansmith/ring/tree/ff0d3d78fcf06151f0cada9d883080ecf9c2e496#versioning--stability) - this should be reconsidered for a replacement if we go further along with this project.

Reply:
Ok, I'll take a note.